community-operators/cluster-logging/clusterlogging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,417 @@
+        rules:
+        - apiGroups:
+          - scheduling.k8s.io
+          resources:
+          - priorityclasses
+          verbs:
+          - "*"
+        - apiGroups:
+          - oauth.openshift.io
+          resources:
+          - oauthclients
+          verbs:
+          - "*"
+      deployments:
+      - name: cluster-logging-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: cluster-logging-operator
+          template:
+            metadata:
+              labels:
+                name: cluster-logging-operator
+            spec:
+              serviceAccountName: cluster-logging-operator
+              containers:
+              - name: cluster-logging-operator
+                image: quay.io/openshift/cluster-logging-operator:latest
+                imagePullPolicy: IfNotPresent
+                command:
+                - cluster-logging-operator
+                env:
+                  - name: WATCH_NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
+                  - name: OPERATOR_NAME
+                    value: "cluster-logging-operator"
+                  - name: ELASTICSEARCH_IMAGE
+                    value: "docker.io/openshift/origin-logging-elasticsearch5:latest"
+                  - name: FLUENTD_IMAGE
+                    value: "docker.io/openshift/origin-logging-fluentd:latest"
+                  - name: KIBANA_IMAGE
+                    value: "docker.io/openshift/origin-logging-kibana5:latest"
+                  - name: CURATOR_IMAGE
+                    value: "docker.io/openshift/origin-logging-curator5:latest"
+                  - name: OAUTH_PROXY_IMAGE
+                    value: "docker.io/openshift/oauth-proxy:latest"
+                  - name: RSYSLOG_IMAGE
+                    value: "docker.io/viaq/rsyslog:latest"
+      - name: elasticsearch-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: elasticsearch-operator
+          template:
+            metadata:
+              labels:
+                name: elasticsearch-operator
+            spec:
+              serviceAccountName: elasticsearch-operator
+              containers:
+                - name: elasticsearch-operator
+                  image: quay.io/openshift/elasticsearch-operator:latest
+                  imagePullPolicy: IfNotPresent
+                  command:
+                  - elasticsearch-operator
+                  ports:
+                  - containerPort: 60000
+                    name: metrics
+                  env:
+                    - name: WATCH_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.namespace
+                    - name: OPERATOR_NAME
+                      value: "elasticsearch-operator"
+  maturity: alpha
```

Comment:
Maturity should be omitted since a process to get these values has not yet been defined.

Ref: https://github.com/operator-framework/operator-marketplace/blob/master/docs/marketplace-required-csv-annotations.md

Reply:
removed maturity